unsigned int* is not the same as unsigned long* on all platforms, our compiler complains about it.
This fixes it, but should probably be talked about with upstream.

--- a/mailsync/Vendor/mailcore2/src/core/zip/MiniZip/zip.c	2026-02-22 19:13:42.129006253 +0100
+++ b/mailsync/Vendor/mailcore2/src/core/zip/MiniZip/zip.c	2026-02-22 19:23:15.753107071 +0100
@@ -1246,7 +1246,7 @@
         unsigned char bufHead[RAND_HEAD_LEN];
         unsigned int sizeHead;
         zi->ci.encrypt = 1;
-        zi->ci.pcrc_32_tab = get_crc_table();
+        zi->ci.pcrc_32_tab = (unsigned long*) get_crc_table();
         /*init_keys(password,zi->ci.keys,zi->ci.pcrc_32_tab);*/

         sizeHead=crypthead(password,bufHead,RAND_HEAD_LEN,zi->ci.keys,zi->ci.pcrc_32_tab,crcForCrypting);
